_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -1927,6 +1927,13 @@ public static function get_updateable_data_list( $selector = '' ) {
 				'validate_callback'  => __CLASS__ . '::validate_boolean',
 				'jp_group'           => 'wordads',
 			),
+			'wordads_custom_adstxt' => array(
+				'description'        => esc_html__( 'Custom ads.txt entries', 'jetpack' ),
+				'type'               => 'string',
+				'default'            => '',
+				'validate_callback'  => __CLASS__ . '::validate_string',
```

Comment:
I think we should also add:
>'sanitize_callback' => 'sanitize_textarea_field',

Without it, it's possible to save php & script tags in the input. We strip these on output, but it would be good to keep them out of the db to prevent any xss.